### PR TITLE
Firebarrel template

### PR DESCRIPTION
**Pull request by IvanBuljovcic**

### CODE REVIEW

**Comment by 0xdevalias** [comment]
When I ran this on localhost and joined the same room, this is the error I saw:

```
Error:  TwilioError: Participant disconnected because of duplicate identity
    at ParticipantDuplicateIdentityError.TwilioError (http://localhost:3000/static/js/0.chunk.js:274523:23)
    at new ParticipantDuplicateIdentityError (http://localhost:3000/static/js/0.chunk.js:274056:151)
    at createTwilioError (http://localhost:3000/static/js/0.chunk.js:273151:36)
    at TwilioConnection.handleMessage (http://localhost:3000/static/js/0.chunk.js:263949:28)
    at TwilioConnection.emit (http://localhost:3000/static/js/0.chunk.js:115691:5)
    at TwilioConnection._handleMessage (http://localhost:3000/static/js/0.chunk.js:268398:12)
    at WebSocket.<anonymous> (http://localhost:3000/static/js/0.chunk.js:268220:20)
```

File: src/components/templates/Playa/Video/Participant.tsx [hunk 1]
```diff
@@ -149,9 +151,11 @@ const Participant: React.FC<React.PropsWithChildren<ParticipantProps>> = ({
   return (
     <div className="participant">
       {videos}
-      <div className="name" onClick={() => setSelectedUserProfile(user)}>
-        {user.anonMode ? DEFAULT_PARTY_NAME : user.partyName} {detail}
-      </div>
+      {showName && (
+        <div className="name" onClick={() => setSelectedUserProfile(user)}>
```
Comment (denisdimitrov) [inline src/components/templates/Playa/Video/Participant.tsx:155]: You can write a function wrapper with useCallback for this setState

File: src/components/templates/Playa/Video/LocalParticipant.tsx [hunk 2]
```diff
@@ -47,24 +60,61 @@ const LocalParticipant: React.FC<LocalParticipantProps> = ({
...
-        <div className="btn" />
-      </div>
-      <div className="av-controls">
-        <div className="mic" onClick={() => setMic(!mic)}>
-          <div className={`btn ${mic ? "on" : "off"}`} />
-        </div>
-        <div className="camera" onClick={() => setCamera(!camera)}>
-          <div className={`btn ${camera ? "on" : "off"}`} />
+      {showLeave && (
+        <div className="leave" onClick={() => leave()}>
```
Comment (denisdimitrov) [inline src/components/templates/Playa/Video/LocalParticipant.tsx:111]: onClick{leave} should be the same.

File: src/components/templates/Playa/Video/LocalParticipant.tsx [hunk 3]
```diff
@@ -47,24 +60,61 @@ const LocalParticipant: React.FC<LocalParticipantProps> = ({
...
+          onClick={() => setCamera(!camera)}
+          size="lg"
+          icon={!camera ? faVideoSlash : faVideo}
+          color={!camera ? "red" : undefined}
+        />
+      );
+    }
+
+    return (
+      <div className="camera" onClick={() => setCamera(!camera)}>
```
Comment (denisdimitrov) [inline src/components/templates/Playa/Video/LocalParticipant.tsx:95]: The state setters can be wrapped in a function with useCallback

Comment (denisdimitrov) [reply src/components/templates/Playa/Video/LocalParticipant.tsx:95]: These also 2-3 more in the same component.

**Review by denisdimitrov — CHANGES_REQUESTED** [review]

**Pull request merged**